frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -403,14 +418,17 @@
                  [(t locale "workspace.assets.delete") handle-delete-typography]]}]
      (when open?
        [:div.group-list
-        (for [typography (sort-by (comp - :ts) typographies)]
+        (for [typography (sort-by :ts typographies)]
```

Comment:
Is it ok to sort this by time? Other assets are sorted by name (see apply-filters function). Also the filter in "Search assets"  is not working, it does not find any typography.

Reply:
The sorting of the library assets is something we'll need to address further down the line. As it is right now, I think it's better to sort by time to improve the user experience.

Reply:
It's possible; although it should be consistent for all kinds of assets. It seems strange that colors or graphics are sorted by name, and typographies by time.